sql/README.md

```diff
@@ -0,0 +1,84 @@
+# SQL Datastore
+
+`sqlds` allows using arbitrary SQL databases as an IPFS datastore. It defines a set of functionality that can be used by any golang client that satisfies the `database/sql` interface. Included in an implementation for the PostgreSQL database. This datastore is forked from `ipfs/go-ds-sql` with a few modifications, and cleanup to the codebase, consisting of:
+
+* Enable table recreation, migration, and index creation as an optional capability when using PostgreSQL
+* Change testing scheme to ensure that the postgres functionality is tested and subsequently the datastore implementation is tested
+  * Upstream doesn't specifically test the postgres implementation which is likely to introduce bugs.
+* Use the `uber-go/multierr` package to combine errors where appropriatey
+    * In transaction mode if any errors happen, the error returned from the rollback was not returned, instead we combine the error for better visibility
+
+# Usage (Postgres)
+
+## Easy (Automatic)
+
+
+The `postgres` datastore has some helper utility to enable automatic table and index creation, as well as the ability to recreate the table by dropping it, and creating it. The following will make it easy to do this:
```

Comment:
Does this need `RunMigrations` set to true?

Reply:
yes it does, and the code wasn't check for that. good catch